container/conf.d/daemon.conf

```diff
@@ -0,0 +1,18 @@
+[daemon]
+systemd_enable = false
```

Comment:
Add

    drop_privileges = false

So the server runs as root in the container - this fixes the permissions issue
that required running a privileged container.